src/lib/homestore_backend/resync_pg_shard.fbs

```diff
@@ -0,0 +1,25 @@
+namespace homeobject;
+
+table Member {
+   uuid : [ubyte];
+   name : [ubyte];
+}
```

Comment:
priority too.

Reply:
Looks like no one is populating the priority and it is just a dummy field. Is it still needed? BTW, priority is not part of create-pg input param.

Reply:
It could be for nuraft leader election priority

Reply:
but no one is actually setting them in HO, I can include them here for future use.